wls-gui-wahllokalsystem/src/styles/custom.css

```
@@ -10,3 +10,8 @@
 .stimmabgabevermerke-table .dynamic-column {
   min-width: 300px;
 }
+
+.v-btn.v-btn--active:hover > .v-btn__overlay,
+.v-btn.v-btn--active:focus > .v-btn__overlay {
+  background-color: rgba(44, 58, 64, 0.04);
```

Was hältst du davon? Hintergrund ist das ich nicht so gut fand hier eine Farbe frei zu definieren. Ich finde die Farben sollten mit dem Theme zuarbeiten.
```suggestion
  opacity: 0.01;
```

---

ich hatte versucht, mit `--v-theme-primary` oder  `--v-hover-opacity` zu arbeiten, aber nicht geschafft auf die vuetify variablen zuzugreifen, deswegen habe ich die farbe hart gecoded. dein vorschlag geht aber auch, gefällt mir auch besser 👍🏽  [518e5ad](https://github.com/it-at-m/Wahllokalsystem/pull/1873/commits/518e5ad1653fd15424da209b725bf206235b6a3c)